### ISSUE TITLE
Upon dimension powered to 99%, server crashes

### ISSUE THREAD

**Issue by Bidgam** [McJty/RFToolsDimensions#361]
Issue description: Server crashes as soon as dimlet powered to 99%.

Steps to reproduce: Created a dimension with a flower forest biome dimlet.

* Minecraft: 1.16.5
* Forge: forge-1.16.5-36.1.32
* McJtyLib: mcjtylib-1.16-5.0.21
* RFTools Dimensions: rftoolsdim-1.16-7.0.10-beta

Crash Log: https://paste.enginehub.org/g37-IwwpJ

I'm able to log back in, but I haven't risked actually trying to enter the dimension as it shows a -1 power, so I don't think it's created. I can remove the dimlet from the device. I started over and it crashed again.

Thanks for taking a look

**Comment by Bidgam** [comment]
Seems to be occurring with any combination of dimlets (so far from what I'm testing). Worked fine with the version of RF Dimensions previous.

**Comment by Bidgam** [comment]
I haven't had a chance to test it yet, but I did not update RFTools Power to the latest version (3.0.9) with the mojang mappings, but I did update the other mods to the mappings version as well.

The server didn't complain or show any dependence, but if this is an obvious mismatch internally still, I can update it. Since the powering of the dimension failed, this sort of tracks. Would be nice to get a programmer confirmation out of curiosity.

Will try in a couple of days anyway.

Thanks
